testplan/exporters/testing/pdf/renderers/entries/assertions.py

```diff
@@ -1,12 +1,9 @@
 import re
 import functools
 
-try:
-    from html import escape as _orig_escape
+from html import escape as _orig_escape
 
-    escape = functools.partial(_orig_escape, quote=False)
-except ImportError:
-    from cgi import escape
+escape = functools.partial(_orig_escape, quote=False)
```

Comment:
Not worth to override it for a single use

Reply:
Unlike other 2 ``escape``, in this file ``escape`` is used many times, I renamed this wrapped to ``html_escape``